### PR TITLE
replace github.com/satori/go.uuid with github.com/google/uuid

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
https://github.com/kubeedge/kubeedge/pull/2355 Unify use google uuid library, but in `github.com/kubeedge/beehive` package, we still use github.com/satori/go.uuid, so just replace it with github.com/google/uuid to keep the same with main repo.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -683,7 +683,6 @@ github.com/russross/blackfriday v0.0.0-20170610170232-067529f716f4/go.mod h1:JO/
 github.com/russross/blackfriday v1.5.2/go.mod h1:JO/DiYxRf+HjHt06OyowR9PTA263kcR/rfWxYHBV53g=
 github.com/russross/blackfriday/v2 v2.0.1/go.mod h1:+Rmxgy9KzJVeS9/2gXHxylqXiyQDYRxCVz55jmeOWTM=
 github.com/ryanuber/columnize v0.0.0-20160712163229-9b3edd62028f/go.mod h1:sm1tb6uqfes/u+d4ooFouqFdy9/2g9QGwK3SQygK0Ts=
-github.com/satori/go.uuid v1.2.0 h1:0uYX9dsZ2yD7q2RtLRtPSdGDWzjeM3TbMJP9utgA0ww=
 github.com/satori/go.uuid v1.2.0/go.mod h1:dA0hQrYB0VpLJoorglMZABFdXlWrHn1NEOzdhQKdks0=
```
Comment (fisherxu) [inline go.sum:686]: This line is no needed?

Comment (gy95) [reply go.sum:686]: yes, this is done automatively by running `hack/update-vendor.sh`

Comment (fisherxu) [reply go.sum:686]: We can try to delete it and run the vendor check

Comment (gy95) [reply go.sum:686]: I tried to delete line 686 `github.com/satori/go.uuid v1.2.0/go.mod h1:dA0hQrYB0VpLJoorglMZABFdXlWrHn1NEOzdhQKdks0=`
But after running `hack/update-vendor.sh`, this deleted line will be reverted back
```
root@guoyao-dev:~/go/src/github.com/kubeedge/kubeedge# git diff
diff --git a/go.sum b/go.sum
index de9fc2f5d..f2832f206 100644
--- a/go.sum
+++ b/go.sum
@@ -683,7 +683,6 @@ github.com/russross/blackfriday v0.0.0-20170610170232-067529f716f4/go.mod h1:JO/
 github.com/russross/blackfriday v1.5.2/go.mod h1:JO/DiYxRf+HjHt06OyowR9PTA263kcR/rfWxYHBV53g=
 github.com/russross/blackfriday/v2 v2.0.1/go.mod h1:+Rmxgy9KzJVeS9/2gXHxylqXiyQDYRxCVz55jmeOWTM=
 github.com/ryanuber/columnize v0.0.0-20160712163229-9b3edd62028f/go.mod h1:sm1tb6uqfes/u+d4ooFouqFdy9/2g9QGwK3SQygK0Ts=
-github.com/satori/go.uuid v1.2.0/go.mod h1:dA0hQrYB0VpLJoorglMZABFdXlWrHn1NEOzdhQKdks0=
 github.com/sean-/seed v0.0.0-20170313163322-e2103e2c3529/go.mod h1:DxrIzT+xaE7yg65j358z/aeFdxmN0P9QXhEzd20vsDc=
 github.com/seccomp/libseccomp-golang v0.9.1 h1:NJjM5DNFOs0s3kYE1WUOr6G8V97sdt46rlXTMfXGWBo=
 github.com/seccomp/libseccomp-golang v0.9.1/go.mod h1:GbW5+tmTXfcxTToHLXlScSlAvWlF4P2Ca7zGrPiEpWo=
root@guoyao-dev:~/go/src/github.com/kubeedge/kubeedge# git commit -a -m 'c'
[uuid b06c3db94] c
 1 file changed, 1 deletion(-)
root@guoyao-dev:~/go/src/github.com/kubeedge/kubeedge# hack/update-vendor.sh 
~/go/src/github.com/kubeedge/kubeedge/staging/src/github.com/kubeedge/beehive ~/go/src/github.com/kubeedge/kubeedge
running 'go mod tidy' for beehive
running 'go mod vendor' for beehive
~/go/src/github.com/kubeedge/kubeedge
~/go/src/github.com/kubeedge/kubeedge/staging/src/github.com/kubeedge/viaduct ~/go/src/github.com/kubeedge/kubeedge
running 'go mod tidy' for viaduct
running 'go mod vendor' for viaduct
~/go/src/github.com/kubeedge/kubeedge
running 'go mod tidy' for repo root
running 'go mod vendor' for repo root
root@guoyao-dev:~/go/src/github.com/kubeedge/kubeedge# git status
On branch uuid
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   go.sum

no changes added to commit (use "git add" and/or "git commit -a")
root@guoyao-dev:~/go/src/github.com/kubeedge/kubeedge# git diff
diff --git a/go.sum b/go.sum
index f2832f206..de9fc2f5d 100644
--- a/go.sum
+++ b/go.sum
@@ -683,6 +683,7 @@ github.com/russross/blackfriday v0.0.0-20170610170232-067529f716f4/go.mod h1:JO/
 github.com/russross/blackfriday v1.5.2/go.mod h1:JO/DiYxRf+HjHt06OyowR9PTA263kcR/rfWxYHBV53g=
 github.com/russross/blackfriday/v2 v2.0.1/go.mod h1:+Rmxgy9KzJVeS9/2gXHxylqXiyQDYRxCVz55jmeOWTM=
 github.com/ryanuber/columnize v0.0.0-20160712163229-9b3edd62028f/go.mod h1:sm1tb6uqfes/u+d4ooFouqFdy9/2g9QGwK3SQygK0Ts=
+github.com/satori/go.uuid v1.2.0/go.mod h1:dA0hQrYB0VpLJoorglMZABFdXlWrHn1NEOzdhQKdks0=
 github.com/sean-/seed v0.0.0-20170313163322-e2103e2c3529/go.mod h1:DxrIzT+xaE7yg65j358z/aeFdxmN0P9QXhEzd20vsDc=
 github.com/seccomp/libseccomp-golang v0.9.1 h1:NJjM5DNFOs0s3kYE1WUOr6G8V97sdt46rlXTMfXGWBo=
 github.com/seccomp/libseccomp-golang v0.9.1/go.mod h1:GbW5+tmTXfcxTToHLXlScSlAvWlF4P2Ca7zGrPiEpWo=
```
So it seems that line 686 cannot be deleted indeed :)

Comment (fisherxu) [reply go.sum:686]: The `github.com/satori/go.uuid` is still used in another vendor `vendor/go.uber.org/zap`.

/approve

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Pull request merged**